wscript

```diff
@@ -61,19 +61,7 @@ def configure(cfg):
 
     cfg.msg('Setting board to', cfg.options.board)
     cfg.env.BOARD = cfg.options.board
-    board_dict = boards.BOARDS[cfg.env.BOARD].get_merged_dict()
-
-    # Always prepend so that arguments passed in the command line get
-    # the priority.
-    for k in board_dict:
-        val = board_dict[k]
-        # Dictionaries (like 'DEFINES') are converted to lists to
-        # conform to waf conventions.
-        if isinstance(val, dict):
-            for item in val.items():
-                cfg.env.prepend_value(k, '%s=%s' % item)
-        else:
-            cfg.env.prepend_value(k, val)
+    boards.get_board(cfg.env.BOARD).configure(cfg)
```

Comment:
Another minor note: get_board() creates new instances, so when a next functionality hit the Board class, be careful to not create multiple instances of the same board.

A preventive measure would be use a local board (or some other non-conflicting name) local variable here, so that next piece of code doesn't consider calling get() again. Alternative would be to cache the instances behind get().